outputs/email_test.go

```diff
@@ -0,0 +1,166 @@
+package outputs
+
+import (
+	"fmt"
+	"net"
+	"net/smtp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func mockSend(errToReturn error, emailSent *int) (func(string, smtp.Auth, string, []string, []byte) error, *emailRecorder) {
+	r := new(emailRecorder)
+	return func(addr string, a smtp.Auth, from string, to []string, msg []byte) error {
+		*r = emailRecorder{addr, a, from, to, msg}
+		if errToReturn == nil {
+			*emailSent++
+		}
+		return errToReturn
+	}, r
+}
+
+type emailRecorder struct {
+	addr string
+	auth smtp.Auth
+	from string
+	to   []string
+	msg  []byte
+}
+
+func TestEmailOutput_Send(t *testing.T) {
+	testCases := []struct {
+		name               string
+		lookupMXFunc       func(name string) ([]*net.MX, error)
+		emailConfig        *EmailOutput
```

Comment:
property should be named `emailOutput` (after it's type)

Reply:
fixed